content/en/docs/concepts/certificaterequest.md

```diff
@@ -72,170 +97,498 @@ Each ready condition consists of the pair `Ready` - a boolean value, and
+      - "awspcaissuers.awspca.cert-manager.io/*"
+
+
+      # For namespaced issuers, the syntax is similar:
+      - "issuers.cert-manager.io/namespace-with-letsencrypt.letsencrypt"
+      #  <-----> <-------------> <-------------------------> <--------->
+      #    type       group            issuer namespace      issuer name
+
+      # You can also allow any issuer type for namespaced issuers:
+      - "issuers.cert-manager.io/*"
+```
+
+If the approver does not have sufficient permissions to set the `Approved` or
+`Denied` conditions, the CertificateRequest is rejected by the cert-manager
+webhook.
+
+### Reason and message of the Approval API
+
+The `reason` and `message` fields on the `Denied` and `Approved` condition
+are approver-specific, meaning that each approver will set their own. Here
+is the set of guidelines:
+
+| Field     | Guideline                                               |
+| --------- | ------------------------------------------------------- |
+| `reason`  | Machine-readable string identifying the approver        |
+| `message` | Human-readable explanation of why the condition was set |
+
+Here is a few examples of approved CertificateRequests:
+
+```yaml
+kind: CertificateRequest
+status:
+  conditions:
+    - type: Approved
+      status: "True"
+      reason: opa-approver-controller
+      message: Certificate request was approved by opa-approver-controller
+```
+
+> The `opa-approver-controller` is hypothetical and does not exist yet.
+
+```yaml
+kind: CertificateRequest
+status:
+  conditions:
+    - type: Approved
+      status: "True"
+      reason: cert-manager.io
+      message: Certificate request has been approved by cert-manager.io
 ```
 
-If the approver does not have sufficient permissions defined above to set the
-Approved or Denied conditions, the request will be rejected by the cert-manager
-validating admission webhook.
+We can also approve the certificate request using the [kubectl
+cert-manager](https://cert-manager.io/docs/usage/kubectl-plugin/) plugin:
 
-- The RBAC permissions *must* be granted at the cluster scope
-- Namespaced signers are represented by a namespaced resource using the syntax of `<signer-resource-name>.<signer-group>/<signer-namespace>.<signer-name>`
-- Cluster scoped signers are represented using the syntax of `<signer-resource-name>.<signer-group>/<signer-name>`
-- An approver can be granted approval for all namespaces via `<signer-resource-name>.<signer-group>/*`
-- The apiGroup must *always* be `cert-manager.io`
-- The resource must *always* be `signers`
-- The verb must *always* be `approve`, which grants the approver the permissions to set *both* Approved and Denied conditions
+```sh
+kubectl cert-manager approve cert-request-1
+```
 
-An example of signing all `myissuer` signers in all namespaces, and
-`clustermyissuers` with the name `myapp`, in the `my-example.io` group:
+Here is what the approved CertificateRequest looks like:
 
 ```yaml
```

Comment:
We are we deleting this list? I think we should have a concise list, before walking into a worked example.